buildifier/buildifier.go

```diff
@@ -24,11 +24,17 @@ import (
 	"io/ioutil"
 	"os"
 	"runtime"
-	"sort"
 	"strings"
+	"sync"
 
-	build "github.com/bazelbuild/buildifier/core"
-	"github.com/bazelbuild/buildifier/differ"
+	core "github.com/bazelbuild/buildifier/core"
```

Comment:
When the package has a different name from the folder, is a bit misleading from the go perspective. So I was thinking to rename the entire package to `core` or rename the folder to `build`. Either of the two solutions are fine to me.